app/lib/actions/remote_execution/run_proxy_command.rb

```diff
@@ -39,18 +39,56 @@ def exit_status
       end
 
       def live_output
-        output = self.output[:proxy_output].present? ? self.output[:proxy_output] : fetch_output['output']
-        output_records(output)
+        records = connection_messages
```

Comment:
does this mean that failing messages will be always first regardless of timestamp?